testutils/resource_utils.go

```diff
@@ -106,6 +107,45 @@ func VerifySameResourceStatuses(tb testing.TB, actual, expected []resource.Statu
 		sortedExpected[i].LastUpdated = time.Time{}
 	}
 
+	// This deferred function provides more concise output for debugging on failure
```

Comment:
Nice; why'd you add? Just felt that you needed better output?

Reply:
ya, resource.Name looks really ugly in output because of the nested fields. Doing it this way is much clearer